integration-tests/features/steps/stack_analyses_v2.py

```diff
@@ -51,7 +51,7 @@ def post_request(context, ecosystem, manifest, with_user_key, with_valid_user_ke
 
     # set default values
     files = {}
-    data = {}
+    data = {'show_transitive': 'false'}
```

Comment:
@dgpatelgit can you please point me to the code in api server which enables / disabled the transitive data on the basis of this flag

Reply:
> @dgpatelgit can you please point me to the code for v2 in api server which enables / disables the transitive data on the basis of this flag

Its under stack_analyses.py -> _make_backbone_request() below is req. params send to backbone